aptos-move/aptos-sdk-builder/src/common.rs

```diff
@@ -34,7 +34,7 @@ fn quote_type_as_format(type_tag: &TypeTag) -> Format {
             U8 => Format::Bytes,
             Vector(type_tag) => {
                 if type_tag.as_ref() == &U8 {
-                    Format::TypeName("VecBytes".into())
```

Comment:
One comment I have we may want to keep consistent with the types in move repo. If we need to change anything on the parsing of VecBytes in move repo, it would be much easier to use existing types.

Reply:
VecBytes are a hack as far as I can tell. We should probably work with the team to remove them. Also move doesn't really have a notion of "transaction agruments".